src/host/polyfills/releaseNoteContent.ts

```diff
@@ -2,6 +2,18 @@
 // Licensed under the MIT License.
 
 export const extensionHighlights = `[
+    {
+        title: ls\`New default start page when launching Edge DevTools\`,
+        subtitle:
+            ls\`Launch page contains useful links and instructions to help get you started.\`,
+        link: 'https://github.com/microsoft/vscode-edge-devtools/pull/350',
+    },
+    {
+        title: ls\`New landing view for the sidebar when no targets are available\`,
+        subtitle:
+            ls\`The view contains instructions and buttons to help launch a target or configure your project's launch.json file.\`,
+        link: 'https://github.com/microsoft/vscode-edge-devtools/pull/357',
+    },
```

Comment:
I only added the new start page and sidebar view to the release notes since they are the only user-facing changes.  I kept all but two of the previous version's notes due to the lack of user impacting content this release.  The notes from the DevTools are going to be the same from last release.

Let me know if there are any concerns or objections to this plan.

Reply:
Seems reasonable to me!